content/tutorials/Super-lightweight-realtime-subscriptions-with-SSE-and-Ably.textile

```diff
@@ -0,0 +1,223 @@
+  newElement.textContent = text;
+  eventList.appendChild(newElement);
+}
+</script>
+```
+
+Here, we defined the script section, where we defined the @apiKey@ variable, which should be replaced with your Ably API key. We forge ahead to define our Ably channel with a variable named channels. We've added a simple check to see if the API key was in fact replaced. Next, we defined a variable called @url@, which is a template string, feeding in our @apiKey@ and @channels@. Notice that the route of this URL is @/event-stream@, which means we want to listen to the HTTP event streams from the server.
+
+We then feed this URL to our @EventSource@, listening for the @onopen@, @onmessage@ and @onerror@ events. Once we get any of these events, we trigger the @addEvent@ function, which appends a new @li@ element to the initial @ul@ element in the HTML markup.
+
+h2(#setting-up-the-subscriber-with-sse). Setting up the subscriber with SSE
+
+For SSE, let us replace our script tag with the following:
+
+
+```[html]
+<script type="text/javascript">
+
+const apiKey = "INSERT-YOUR-API-KEY-HERE";
+
+// can have multiple channels, comma-separated.
+// If a channel name is not url-safe, URL-encode it first
+const channels = "example-channel";
+
+if(apiKey.indexOf("INSERT") === 0) {
+  alert("Add your API key to example.html");
+  throw 'Cannot proceed without api key';
+}
+
+const url = `https://realtime.ably.io/sse?v=1.1&key=${apiKey}&channels=${channels}`
+const eventSource = new EventSource(url);
+eventSource.onopen = function() {
+  addEvent(`Connected and subscribed to channels: ${channels.split(',').join(', ')}`);
+}
+
+eventSource.onmessage = function(event) {
+  const message = JSON.parse(event.data);
+  addEvent(`Channel: ${message.channel} - Message: ${message.name} - ${message.data}`);
+}
+
+eventSource.onerror = function(error) {
+  if(error.data) {
+    addEvent(`Error: ${error.data}`);
+  } else {
+    addEvent('Error connecting to Ably');
+  }
+}
+
+const eventList = document.querySelector('ul#events');
+
+function addEvent(text) {
+  const newElement = document.createElement("li");
+  newElement.textContent = text;
+  eventList.appendChild(newElement);
+}
+</script>
+```
+
+The code above is pretty similar to that of the event stream. The only difference here is the URL supplied to the @EventSource@. Here, the @EventSource@ connects to the @/sse@ endpoint, as opposed to the @event-stream@ endpoint.
+
+bq. The @/event-stream@ endpoint will give an SSE response if the @Accept@ header is set to @text/event-stream@. The @/sse@ endpoint is also provided as an easier way of forcing an SSE response. 
+
+
+
+h2(#conclusion). Conclusion
+
+
+In this article, we have seen how to subscribe and publish SSE events using Ably. We have seen how to publish SSE events to Ably in three different languages as well as seen the two different types of streams Ably provides. Hope this tutorial has helped you to get started with Ably and HTTP events / SSE?
```

Comment:
'article' -> 'tutorial'

Reply:
'how to subscribe and publish SSE events using Ably' -> 'how to subscribe to real-time events using SSE and Ably'

"We have seen how to publish SSE events to Ably in three different languages as well as seen the two different types of streams Ably provides. Hope this tutorial has helped you to get started with Ably and HTTP events / SSE?"  - we can remove this and add further reading links such as 

- Docs
- Video

Please add them as placeholders for now and I'll add the link afterward.